packages/wmr/package.json

```diff
@@ -0,0 +1,161 @@
+
+{
+	"name": "wmr",
+	"version": "1.1.0",
+	"bin": "wmr.cjs",
+	"type": "module",
+	"scripts": {
+		"demo": "cd demo && node --experimental-modules ../src/cli.js 2>&1",
+		"demo:prod": "cd demo && node --experimental-modules ../src/cli.js build --prerender 2>&1",
+		"demo:serve": "cd demo && node --experimental-modules ../src/cli.js serve",
+		"dev": "cross-env PROFILE=true nodemon -w src --exec \"npm run -s demo\"",
+		"build": "rollup -c",
+		"ci": "yarn build && cd demo && node ../wmr.cjs build --prerender",
+		"prepack": "node --experimental-modules ./src/lib/~publish.js",
+		"test-e2e": "cross-env JEST_PUPPETEER_CONFIG=jest-puppeteer.config.cjs jest",
+		"test": "eslint src test && npm run test-e2e"
+	},
+	"author": "The Preact Authors (https://preactjs.com)",
+	"license": "MIT",
+	"repository": "preactjs/wmr",
+	"files": [
+		"wmr.cjs",
+		"types.d.ts"
+	],
+	"types": "./types.d.ts",
+	"babel": {
+		"plugins": [
+			"@babel/plugin-transform-modules-commonjs"
+		]
+	},
+	"eslintConfig": {
+		"extends": [
+			"developit",
+			"prettier",
+			"plugin:prettier/recommended"
+		],
+		"plugins": [
+			"prettier",
+			"import"
+		],
+		"rules": {
+			"no-console": "off",
+			"prefer-rest-params": "off",
+			"prefer-spread": "off",
+			"no-prototype-builtins": "off",
+			"function-call-argument-newline": [
+				"error",
+				"consistent"
+			],
+			"react/jsx-no-bind": "off",
+			"import/extensions": [
+				"error",
+				"always",
+				{
+					"ignorePackages": true
+				}
+			]
+		},
+		"globals": {
+			"jestPuppeteer": "readonly",
+			"browser": "readonly",
+			"page": "readonly"
+		}
+	},
+	"eslintIgnore": [
+		"wmr.cjs",
+		"test/fixtures/**/*.expected.*",
+		"test/fixtures/*/dist",
+		"test/fixtures/*/.cache"
+	],
+	"prettier": {
+		"singleQuote": true,
+		"trailingComma": "none",
+		"useTabs": true,
+		"printWidth": 120,
+		"arrowParens": "avoid"
+	},
+	"lint-staged": {
+		"{src,test}/**/*.js": [
+			"eslint --fix",
+			"prettier --write"
+		],
+		"*.md": [
+			"prettier --write"
+		]
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "lint-staged"
```

Comment:
Don't have much experience with workspaces, so I might be totally wrong: Wonder if this should be in the root package.json.

Reply:
Oh it is, I still have to do cleanup, looking into the build issues with the wmr.cjs file